### ISSUE TITLE
[🐞] Latest qwik-react version (0.5.2) shipped without TypeScript types

### ISSUE THREAD

**Issue by octet-stream** [BuilderIO/qwik#5816]
### Which component is affected?

Qwik React

### Describe the bug

After updating qwik-react from 0.5.1 to 0.5.2 I got following error:

```
Could not find a declaration file for module '@builder.io/qwik-react/vite'
```

This error affects both `qwik-react` and `qwik-react/vite`
If you open `@builder.io/qwik-react` code in npm you will see that the package lacks `lib/types` directory: https://www.npmjs.com/package/@builder.io/qwik-react?activeTab=code
But the previous version have those: https://www.npmjs.com/package/@builder.io/qwik-react/v/0.5.1?activeTab=code

### Reproduction

https://stackblitz.com/edit/qwik-starter-zsfa6w?file=vite.config.ts

### Steps to reproduce

1. Open `vite.config.ts` and you will see the error at the line 5.
2. Then open `src/integrations/react/mui.tsx` and you'll see TS complaining about missing types on the line 3.

### System Info

```shell
System:
    OS: macOS 14.2.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 58.19 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.3.0 - ~/Library/Caches/fnm_multishells/96117_1704896998443/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 10.2.4 - ~/Library/Caches/fnm_multishells/96117_1704896998443/bin/npm
    pnpm: 8.14.3 - /opt/homebrew/bin/pnpm
    bun: 1.0.25 - /opt/homebrew/bin/bun
  Browsers:
    Chrome: 121.0.6167.139
    Safari: 17.2.1
  npmPackages:
    @builder.io/qwik: 1.4.3 => 1.4.3 
    @builder.io/qwik-auth: 0.1.3 => 0.1.3 
    @builder.io/qwik-city: 1.4.3 => 1.4.3 
    @builder.io/qwik-react: 0.5.2 => 0.5.2 
    undici: ^6.6.0 => 6.6.0 
    vite: 5.0.12 => 5.0.12
```

### Additional Information

The current solution is to stay on 0.5.1.

**Comment by wmertens** [comment]
Argh

**Comment by wmertens** [comment]
published 0.5.3, containing types this time.
